public/locales/auset-te.json

```diff
@@ -0,0 +1,426 @@
+    },
+    "winter": {
+      "name": "శీతాకాలం",
+      "setupTitle": "శీతాకాలపు మ్యాప్‌ని సెటప్ చేయండి",
+      "setup": "<i>1.</i> గేమ్ బాక్స్ నుండి శరదృతువు/శీతాకాలపు మ్యాప్ బోర్డ్‌ను పట్టుకుని, శీతాకాలపు వైపు పైకి కనిపించేలా కింద ఉంచండి.<br/><i>2.</i> 12 సూట్ మార్కర్‌లను సేకరించండి, వాటిని క్రిందికి తిప్పండి మరియు వాటిని షఫుల్ చేయండి, ఆపై ప్రతి క్లియరింగ్‌లో ఒకదాన్ని ఉంచండి.<br/><i>3.</i> \"R\" <i>తో గుర్తించబడిన మ్యాప్‌లోని ప్రతి స్లాట్‌లో ఒక శిధిలాన్ని ఉంచండి ( మొత్తం నాలుగు)</i>.<br/><i>4.</i> ఈ అంశాలను మ్యాప్ ఎగువన ఉన్న ఐటెమ్ సప్లై యొక్క మ్యాచింగ్ స్పేస్‌లపై ఉంచండి: 2 <Boot/>, 2 <Bag/ >, 1 <Crossbow/>, 1 <Hammer/>, 2 <Sword/>, 2 <Tea/>, 2 <Coin/>.<br/><i>5.</i> దగ్గర రెండు పాచికలు ఉంచండి పటము."
+    }
+  },
+  "vagabond": {
+    "adventurer": {
+      "name": "సాహసికుడు",
+      "action": "మెరుగుపరచండి",
+      "effect": "క్వెస్ట్ చర్య తీసుకునేటప్పుడు ఒక్కో మలుపుకు ఒకసారి, మీరు ఒక అయిపోని వస్తువును ఏదైనా ఇతర వస్తువుగా పరిగణించవచ్చు. అన్వేషణను పూర్తి చేయడానికి మీరు దాన్ని ఎగ్జాస్ట్ చేసినప్పుడు, ఆ అంశాన్ని కూడా పాడు చేయండి."
+    },
+    "arbiter": {
+      "name": "మధ్యవర్తి",
+      "action": "రక్షకుడు",
+      "effect": "యుద్ధంలో పాచికలు వేయడానికి ముందు, డిఫెండర్ యుద్ధం యొక్క క్లియరింగ్‌లో ఆర్బిటర్‌ను చేర్చుకోవచ్చు. ఆర్బిటర్ ఒక విక్టరీ పాయింట్‌ని స్కోర్ చేస్తాడు మరియు డిఫెండర్ యొక్క గరిష్ట రోల్డ్ హిట్‌లకు అతని దెబ్బతినని <Sword/> మొత్తాన్ని జోడిస్తుంది. మధ్యవర్తి తనను తాను చేర్చుకోలేరు లేదా తనకు వ్యతిరేకంగా నమోదు చేసుకోలేరు."
+    },
+    "harrier": {
+      "name": "హారియర్",
+      "action": "నెమ్మదిగా కదలు",
+      "effect": "మీ వాగాబాండ్ బంటుని <i>(ఇతర ముక్కలు కాదు)</i>ని ఏ <Boot/> అయిపోకుండా మ్యాప్‌లోని ఏదైనా క్లియరింగ్ <i>(విరోధి కూడా)</i>కి తరలించడానికి <టార్చ్/>ని ఎగ్జాస్ట్ చేయండి."
+    },
+    "ranger": {
+      "name": "రేంజర్",
+      "action": "సురక్షితమైన దాక్కునేచోటు",
+      "effect": "మూడు వస్తువులను రిపేర్ చేయడానికి ఒక <Torch/> ఖాళీ చేయండి. తర్వాత, వెంటనే డేలైట్‌ని ముగించి సాయంత్రం ప్రారంభించండి."
+    },
+    "ronin": {
+      "name": "రోనిన్",
+      "action": "వేగంగా దెబ్బవేయు",
+      "effect": "యుద్ధంలో <i>(రోలింగ్ తర్వాత)</i> అదనపు హిట్‌ని ఎదుర్కోవడానికి మీరు <Sword/>ని ఖాళీ చేయవచ్చు."
+    },
+    "scoundrel": {
+      "name": "దుష్టుడు",
+      "action": "మండుతున్న భూమి",
+      "effect": "<Torch/>ని ఖాళీ చేసి, దానిని మీ క్లియరింగ్‌లో ఉంచండి. ఆ క్లియరింగ్ నుండి అన్ని శత్రువు ముక్కలను తొలగించండి. <Torch/>తో ముక్కలను ఉంచడం లేదా క్లియరింగ్‌లోకి తరలించడం సాధ్యం కాదు. <i>(మీరు ఈ క్లియరింగ్‌లో ఉంటారు. మీరు బయటకు వెళ్లిన తర్వాత, మీరు తిరిగి లోపలికి వెళ్లలేరు. \"ఫేవర్ ఆఫ్ ది...\" కార్డ్‌తో <Torch/> తీసివేయబడదు ఎందుకంటే ఇది శత్రువు ముక్క కాదు. )</i>"
+    },
+    "thief": {
+      "name": "దొంగ",
+      "action": "దొంగిలించు",
+      "effect": "మీ క్లియరింగ్‌లో ఉన్న ఏ ఆటగాడు నుండి అయినా యాదృచ్ఛిక కార్డ్‌ని తీసుకోవడానికి ఒక <Torch/>ని ఖాళీ చేయండి."
+    },
+    "tinker": {
+      "name": "లోహపు పాత్రలను బాగు చేయువాడు",
```

Comment:
Ideally the vagabond name should just be a single word that could describe someone who creates. A literal translation of "tinker" is unnecessary and not required for the gameplay. Conciseness is far more important than accuracy here, you just want a fun word that people might associate with how this vagabond plays.